src/components/is-scrolling-up.mixin.ts

```diff
@@ -0,0 +1,38 @@
+import { ScrollDirection, XOn } from '@empathyco/x-components';
+import Vue from 'vue';
+import Component from 'vue-class-component';
+@Component
+export default class IsScrollingUp extends Vue {
+  protected stopsScrollDown = true;
+  protected isScrollingUp = true;
+  protected scrollOffset = 200;
+
+  @XOn('UserChangedScrollDirection')
+  scrollDirectionChanged(direction: ScrollDirection): void {
+    this.stopsScrollDown = direction === 'UP';
+  }
```

Comment:
This will listen to any scroll direction change managed by the scroll module. We should be concerned only by changes on the main scroll. Change the implementation to using the state of the main scroll only.

```suggestion
  @State('scroll', 'data')
  public scrollPositionsMap!: Dictionary<ScrollComponentState>;

  protected get mainScrollPosition(): number {
    return this.scrollPositionsMap['main-scroll']?.position;
  }
```